### PR TITLE
config is not in /etc/georchestra/datadir (2)

### PR DESCRIPTION
**Pull request by MaxiReglisse**
sorry, new PR because i could not edit the file any more...

### CODE REVIEW

File: docs/docker.md [hunk 1]
```diff
@@ -21,7 +21,8 @@ Feel free to comment out the apps you do not need.
 Create a configuration directory on your host machine:
 ```
 git clone -b docker-master https://github.com/georchestra/datadir.git
-sudo mv datadir /etc/georchestra
+sudo mkdir /etc/georchestra
+sudo mv datadir/* /etc/georchestra/.
```
Comment (fvanderbiest) [inline docs/docker.md]: I dislike this command since it fails to copy the `.git` folder, which is useful for those who will want to version their datadir.

Comment (MaxiReglisse) [inline docs/docker.md]: i understand... a solution could consist in downloading files in the right directory, like that:
sudo cd /etc
sudo git clone -b docker-master https://github.com/georchestra/datadir.git georchestra
if the georchestra directory already exists, the download is stopped, with this warning:
fatal: le chemin de destination 'georchestra' existe déjà et n'est pas un répertoire vide.
hint:
you should have a git prompt if you want to know the branch on which you are.
cat /root/.bashrc
# http://stackoverflow.com/questions/15883416/adding-git-branch-on-the-bash-command-prompt
source /usr/local/bin/git-prompt.sh
PS1='[\u@\h \W$(__git_ps1 " (%s)")]\$ '

Comment (MaxiReglisse) [inline docs/docker.md]: sorry for the size of the link... there is a # just before http.

**Comment by fvanderbiest** [comment]
Fixed with https://github.com/georchestra/georchestra/commit/4b663b118753482af1e01f9dbd5deaa038ea0fd1

**Pull request closed without merging**